Makefile

```diff
@@ -1,9 +1,9 @@
-ZBAR_SOURCE = zbar-0.10
+ZBAR_SOURCE = zbar-0.23.90
```

Comment:
I was thinking about using the git submodule points to https://github.com/mchehab/zbar/tree/0.23.92

Can you help to do this? Thanks!

Reply:
Gave it a quick try, no instant joy with https://github.com/mchehab/zbar/tree/0.23.92 but success with https://linuxtv.org/downloads/zbar/zbar-0.23.90.tar.gz.

Although the archives have the same version, they are quite different in some respect.

Will adapt the `Makefile`.